### PR TITLE
Use adminCode stats analysis requests COUNTRY=mozambique

### LINKED ISSUE #631: Exposure analysis and report use incorrect admin attributes in the table view
When I run an exposure analysis and export a report of that analysis, the table that is generated uses the incorrect admin attributes for the name of the area.  

Steps to reproduce:
  1. REACT_APP_COUNTRY=mozambique
  2. Open the tropical storm layer
  3. Go to date 2022-03-12
  4. Click the 'exposure analysis' button
  5. Toggle 'table view' on
  6. Select 'export' and then 'report' 

Here's a screenshot of the current values in the table view in the app which are meaningless to users. It's also missing the total row.
Screen Shot 2022-11-08 at 21 47 48

And here's a screenshot of the values seen in the report. Note, the total row is also missing commas in the report
Screen Shot 2022-11-08 at 21 48 32

Please use the same approach that is used in the 'run analysis' output table. The header should say 'Name', and the values in the first column should be the name of the admin area, using commas to separate admin levels. See screenshot below for an example of a run analysis output in Mozambique:
Screen Shot 2022-11-08 at 21 45 45

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
**Related issue** Closes #631 

**How to test the feature**
- Select Tropical Storm layer.
- Choose 25-01-2022 as date.
- Click on Exposure Analysis button.

**Screenshot/video of feature**
image

### CODE REVIEW

**Comment by wadhwamatic** [comment]
@JorgeMartinezG - this looks better, thanks! But instead of using the variable name of the admincode, why not use the simple header 'Name' like we have in the run analysis result?

I noticed that the flood exposure report is no longer available after running a flood exposure analysis. I'm not sure if this PR broke this or if something else did.

**Comment by wadhwamatic** [comment]
@JorgeMartinezG - this looks good for Cambodia (though there's another issue I'll open separately). I couldn't test it for Mozambique as GeoServer requests are timing out. I'll try again tomorrow.

**Comment by JorgeMartinezG** [comment]
@wadhwamatic added a commit which closes #668

File: frontend/src/components/MapView/utils.ts [hunk 1]
```diff
@@ -77,13 +80,16 @@ export const convertToTableData = (result: ExposedPopulationResult) => {
     ? uniq(features.map(f => f.properties && f.properties[key]))
     : [statistic];
 
+  const { adminLevelNames } = getBoundaryLayerSingleton();
+  const lowestLevel = adminLevelNames[adminLevelNames.length - 1];
```
Comment (ericboucher) [inline frontend/src/components/MapView/utils.ts:84]: I think this appears in a few other places, can you create a util function for this?

File: frontend/src/utils/analysis-utils.ts [hunk 2]
```diff
@@ -835,3 +835,27 @@ export function downloadCSVFromTableData(
 // type of results that have the tableData property
 // and are displayed in the left-hand "RUN ANALYSIS" panel
 export type TabularAnalysisResult = BaselineLayerResult | PolygonAnalysisResult;
+
+export const appendBoundaryProperties = (
```
Comment (ericboucher) [inline frontend/src/utils/analysis-utils.ts:839]: Can you please add a docstring to this function?

**Review by ericboucher — APPROVED** [review]
Left a few comments but looks good otherwise, thanks.

**Comment by JorgeMartinezG** [comment]
PR is ready for merging @wadhwamatic cc @ericboucher

**Comment by wadhwamatic** [comment]
@JorgeMartinezG - I added a minor change to capitalize table headers

**Pull request merged**